.env.example

```
@@ -0,0 +1,11 @@
+# If you are running this on a different machine, set the following env variables.
+AUGMENTOS_API_KEY=
+
+# When the SDK is updated to include settings, this won't be needed since it will be set automatically per user based on where their client is pointng.
+CLOUD_HOST_NAME= # e.g. prod.augmentos.org
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)